Makefile

```
@@ -141,6 +154,7 @@ bip-orchestrate-vm-apply-patches: bip-orchestrate-vm
 bip-orchestrate-vm-patched: bip-orchestrate-vm-apply-patches
 
 .PHONY: lifecycle-agent
+# Checkout lifecycle-agent from LCA_GIT_BRANCH for local usage
 lifecycle-agent:
```

checkmake (0.3.2)

[warning] Target body for "lifecycle-agent" exceeds allowed length of 5 lines (9).

(maxbodylength)